app/controllers/auth/AuthAction.scala

```diff
@@ -43,55 +44,49 @@ class AuthActionImpl @Inject()(
   configDecorator: ConfigDecorator)(implicit ec: ExecutionContext)
     extends AuthAction with AuthorisedFunctions {
 
-  override def invokeBlock[A](request: Request[A], block: AuthenticatedRequest[A] => Future[Result]): Future[Result] = {
+  object GT100 {
+    def unapply(confLevel: ConfidenceLevel): Option[ConfidenceLevel] =
+      if (confLevel.level > ConfidenceLevel.L100.level) Some(confLevel) else None
+  }
 
-    val compositePredicate = CredentialStrength(CredentialStrength.strong) and ConfidenceLevel.L200
+  object LT200 {
+    def unapply(confLevel: ConfidenceLevel): Option[ConfidenceLevel] =
+      if (confLevel.level < ConfidenceLevel.L200.level) Some(confLevel) else None
+  }
+
+  val weak = CredentialStrength.weak
+  val strong = CredentialStrength.strong
+
+  override def invokeBlock[A](request: Request[A], block: AuthenticatedRequest[A] => Future[Result]): Future[Result] = {
 
     implicit val hc: HeaderCarrier =
       HeaderCarrierConverter.fromHeadersAndSession(request.headers, Some(request.session))
-    authorised(compositePredicate)
+    authorised()
       .retrieve(
         Retrievals.nino and
+          Retrievals.affinityGroup and
           Retrievals.allEnrolments and
           Retrievals.credentials and
+          Retrievals.credentialStrength and
           Retrievals.confidenceLevel and
           Retrievals.name and
           Retrievals.loginTimes and
           Retrievals.trustedHelper and
           Retrievals.profile) {
 
-        case nino ~ Enrolments(enrolments) ~ Some(credentials) ~ confidenceLevel ~ name ~ logins ~ Some(trustedHelper) ~ profile =>
-          val trimmedRequest: Request[A] = request
-            .map {
-              case AnyContentAsFormUrlEncoded(data) =>
-                AnyContentAsFormUrlEncoded(data.map {
-                  case (key, vals) => (key, vals.map(_.trim))
-                })
-              case b => b
-            }
-            .asInstanceOf[Request[A]]
+        case _ ~ Some(Individual) ~ _ ~ _ ~ (Some(`weak`) | None) ~ _ ~ _ ~ _ ~ _ ~ _ =>
```

Comment:
If you update `auth-client` you can change it to this
```suggestion
        case _ ~ Some(Individual) ~ _ ~ _ ~ (Some(CredentialStrench.weak) | None) ~ _ ~ _ ~ _ ~ _ ~ _ =>
```